batch/enrichment/src/main/java/HashMapEnrichment.java

```diff
@@ -41,37 +37,31 @@
  * <p>
  * The {@link HashJoinP} expects enrichment table on input ordinal 0 and items
  * to enrich on all other ordinals. The edge at ordinal 0 must have {@link
- * Edge#priority(int) priority} set to -1 to ensure, that items on this edge
- * are processed before items to enrich.
+ * com.hazelcast.jet.Edge#priority(int) priority} set to -1 to ensure, that
+ * items on this edge are processed before items to enrich.
  * <p>
  * The {@code readTickerInfoMap} reads the items in distributed way. In order
- * to have full copy on each member we need to broadcast and distribute to a
- * processor with local parallelism of 1. From here, we partition the table to
- * local processors, so that each can process part of the data. Items from
- * "Trades source" to "Joiner" are partitioned locally by the same key.
+ * to have full copy on each node we need to broadcast and distribute to a
+ * processor with local parallelism of 1. From there, we distribute the same
+ * {@code Map} instance to all local processors. Sending locally does not copy
+ * the object, thus this will not increase memory usage.
  * <p>
  * The DAG is as follows:
  * <pre>{@code
- *                                    +--------------------+
- *                                    | Read tickerInfoMap |
- *                                    +---------+----------+
- *                                              |
- *                                              | TickerInfo
- *                                              |  (broadcast, distributed edge)
- *                                      +-------+-------+
- *                                      | Pass through  |
- *                                      +-------+-------+ (localParallelism = 1)
- *                                              |
- * +-----------------+                          | TickerInfo
- * |  Trades source  |                          |  (local partitioned edge)
- * +--------+--------+                 +--------+--------+
- *          |                          |  Squash to map  |
- *          | Trade                    +--------+--------+
- *          |  (local partitioned edge)         |
- * +--------+--------+                          | Map<ticker, tickerInfo>
- * |      Joiner     +--------------------------+  (one-to-one, priority edge)
- * +--------+--------+
- *          |
+ *                             +--------------------+
+ *                             | Read tickerInfoMap |
+ *                             +---------+----------+
+ *                                       |
+ * +-----------------+                   | TickerInfo
+ * |  Trades source  |                   |  (broadcast, distributed edge)
+ * +--------+--------+          +--------+--------+
+ *          |                   |  Squash to map  |
+ *          | Trade             +---------------+-+    (localParallelism = 1)
+ *          |  (local round+robin edge)         |
```

Comment:
We currently call this edge type "variable unicast". "Round-robin" has been discouraged as confusing/ambiguous. Simply "local edge" should be enough here.